lib/Doctrine/ruleset.xml

```diff
@@ -311,6 +311,8 @@
     <rule ref="SlevomatCodingStandard.PHP.UselessParentheses"/>
     <!-- Forbid useless semicolon `;` -->
     <rule ref="SlevomatCodingStandard.PHP.UselessSemicolon"/>
+    <!-- Require /* @var type $foo */ and similar simple inline annotations to be replaced by assert() -->
```

Comment:
Does this sniff support `/* @var ArrayIterator $iterator */` with one `*` too like in this comment? IDEs like PHPStorm support those too, even though it doesn't autocomplete on "/*<space>". In case it supports the single-`*`, some tests for those would be nice.

Reply:
It does not.  However it is reported (and fixed) by https://github.com/doctrine/coding-standard/blob/e008cac4f4100f23361b8c7c31a18a268be66ad2/lib/Doctrine/ruleset.xml#L200

Reply:
Is it possible that both can be used? The first one fixing a `/*` and the other one handles it for the `assert()`?

Reply:
A CS rule enforcing `/**` style would be preferable: please note that current static analysis tools fail to parse `/*` AFAIK

Reply:
`InvalidDocCommentDeclaration` fixes `/*` to `/**` in first round and in second round `RequireExplictionAssertion` changes doccomment to assertion :)

Reply:
> A CS rule enforcing `/**` style would be preferable: please note that current static analysis tools fail to parse `/*` AFAIK

Last time I used NetBeans (granted, a while ago), it only supported vardoc syntax (`/*`) but not */**`.

Reply:
That changed a long time ago: at the moment, the ecosystem is pushing for `/**`, and enabling that sniff could put a tombstone on this

Reply:
@Ocramius Not sure I undrestand you well but the `InvalidInlineDocCommentDeclaration` is already enabled ;)

Reply:
@kukulich yeah, this was mostly for @DASPRiD :+1: